taxonomies/categories.txt

```diff
@@ -40355,23 +41031,27 @@ ro:Cod, Cozi
 
 <en:Cods
 en:Cod livers
+de:Kabeljaulebern
 fi:turskanmaksat
 fr:Foies de morue, foie de morue
 nl:Kabeljauwlevers
 
 <en:Cod livers
 en:Cod Liver Oils
+fr:Huiles de foie de morue
```

Comment:
`ERROR - fr:huiles-de-foie-de-morue already is a synonym of huiles-de-foie-de-cabillaud (en:cod-liver-oils) - cannot make a synonym of huiles-de-foie-de-morue for en:cod-liver-oils`

Reply:
@stephanegigandet